rootconf/default/etc/init.d/dropbear.sh

```diff
@@ -14,31 +15,18 @@ daemon=/usr/sbin/$name
 
 [ -x $daemon ] || exit 0
 
-ARGS="-m -B"
+ARGS="-m -B -R"
```

Comment:
Yeah I'll remove that, initially I though i could rework a bit around it instead using get_keys() and since we must generate keys on first start and save them to env